content/en/docs/tasks/debug/debug-cluster/troubleshoot-kubectl.md

```diff
@@ -0,0 +1,144 @@
+---
+reviewers:
+# Should be added by an experienced contributor
+title: "Troubleshooting kubectl"
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+This documentation is about investigating and diagnosing
+{{<glossary_tooltip text="kubectl" term_id="kubectl">}} related issues.
+If you are unable to access `kubectl` or unable to connect to your cluster using
+`kubectl`, this document describes several common scenarios and potential solutions
+to identify and address the likely cause.
+
+<!-- body -->
+
+## {{% heading "prerequisites" %}}
+
+* You need to have a Kubernetes cluster.
+* You also need to have `kubcectl` installed.
+
+## Verify kubectl setup
+
+Make sure you have installed and configured `kubectl` correctly on your local machine.
+Check the `kubectl` version to ensure it is up-to-date and compatible with your cluster.
+
+Check kubectl version:
+
+```shell
+kubectl version
+```
+
+You'll see a similar output:
+
+```console
+Client Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.4",GitCommit:"fa3d7990104d7c1f16943a67f11b154b71f6a132", GitTreeState:"clean",BuildDate:"2023-07-19T12:20:54Z", GoVersion:"go1.20.6", Compiler:"gc", Platform:"linux/amd64"}
+Kustomize Version: v5.0.1
+Server Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.3",GitCommit:"25b4e43193bcda6c7328a6d147b1fb73a33f1598", GitTreeState:"clean",BuildDate:"2023-06-14T09:47:40Z", GoVersion:"go1.20.5", Compiler:"gc", Platform:"linux/amd64"}
+
+```
+
+If you see `Unable to connect to the server: dial tcp <server-ip>:8443: i/o timeout`,
+instead of `Server Version`, you need to troubleshoot kubectl connectivity with your cluster.
+
+If you have installed the `kubectl` but your shell gives the `command not found` error
+make sure that the `$PATH` environment variable is correctly configured and you have
+installed the kubectl by following the [official documentation for installing kubectl](/docs/tasks/tools/#kubectl).
```

Comment:
Make sure A and B ...
The implicit assumption is that A should happen before B, right?
How about swapping the A and B above?